torch/_torch_docs.py

```diff
@@ -3152,7 +3152,24 @@ def merge_dicts(*dicts):
            r"""
 flatten(input, start_dim=0, end_dim=-1) -> Tensor
 
-Flattens a contiguous range of dims in a tensor.
+By default, flattens :attr:`input` into a one-dimensional tensor with the same number of elements.
+If :attr:`start_dim` or :attr:`end_dim` are passed, only dimensions included in the range are
+flattened.
+
+Flattening an :attr:`input` with shape :math:`(l*, f+, r*)` produces an output shape of
+:math:`(l*, \prod f+, r*)` where :math:`l*` is the sizes of the zero or more "left" dimensions,
+:math:`f+` is the sizes of the one or more dimensions between :attr:`start_dim` and :attr:`end_dim` that
```

Comment:
"between" is tricky here because it implies that start_dim and end_dim are not in `f+`, but they are

Reply:
Ahh, but "starting with :attr:`start_dim` and ending with :attr:`end_dim`" is so wordy